### PR TITLE
Updated version of ember-pikaday-shim

### PR DESCRIPTION
**Pull request by juwara0**
### This project uses [semver](semver.org), please check the scope of this pr:

- [ ] #none# - documentation fixes and/or test additions
- [X] #patch# - backwards-compatible bug fix
- [ ] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change

Closes: https://github.com/ciena-frost/ember-frost-date-picker/issues/81

# CHANGELOG
* **Updated** to latest version of `ember-pikaday-shim` which now includes `pikaday` and `ember-cli-moment-shim` in its dependencies.
* **Removed** no longer needed `pikaday`
* **Removed** no longer needed `ember-cli-moment-shim`
* **Updated** blueprints to no longer include dependencies being provided by `ember-pikaday-shim`
* **Updated** `index.js` to correctly bind context to call of this._super.included()

### CODE REVIEW

**Comment by notmessenger** [comment]
👍

**Comment by notmessenger** [comment]
Closes https://github.com/ciena-frost/ember-frost-date-picker/issues/81

File: index.js [hunk 1]
```diff
@@ -19,25 +19,9 @@ module.exports = {
     /* eslint-enable no-unused-expressions */
   },
 
-  _findHost: function () {
```
Comment (quincyle) [inline index.js]: I know this code is specifically added to support using this addon in ember-engine. Can you confirm that the import still works with the latest engine?

Comment (juwara0) [reply index.js]: @quincyle - I did not test this with an ember-engine. I did however, test this change with an app and a nested add-on.

Please let me know if this does not work with your ember-engine.

**Pull request merged**